test/Datadog.Trace.ClrProfiler.IntegrationTests/Elasticsearch7Tests.cs

```diff
@@ -0,0 +1,161 @@
+// <copyright file="Elasticsearch7Tests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+#if !NET452
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class Elasticsearch7Tests : TestHelper
+    {
+        public Elasticsearch7Tests(ITestOutputHelper output)
+            : base("Elasticsearch.V7", output)
+        {
+            SetServiceVersion("1.0.0");
+            SetCallTargetSettings(true);
+        }
+
+        public static IEnumerable<object[]> GetElasticsearch() => PackageVersions.ElasticSearch7;
+
+        [Theory]
+        [MemberData(nameof(GetElasticsearch))]
+        [Trait("Category", "EndToEnd")]
+        public void SubmitsTraces(string packageVersion)
+        {
+            int agentPort = TcpPortProvider.GetOpenPort();
+
+            using (var agent = new MockTracerAgent(agentPort))
+            using (var processResult = RunSampleAndWaitForExit(agent.Port, packageVersion: packageVersion))
+            {
+                Assert.True(processResult.ExitCode == 0, $"Process exited with code {processResult.ExitCode}");
+
+                var expected = new List<string>();
+
+                // commands with sync and async
+                for (var i = 0; i < 2; i++)
+                {
+                    expected.AddRange(new List<string>
+                    {
+                        "Bulk",
+                        "Create",
+                        "Search",
+                        "DeleteByQuery",
```

Comment:
Are these the same in all versions of ES (Might be nice to extract them and be explicit if so)

Reply:
They're different sample projects so I don't think there's value in de-duplicating the list of spans

Reply:
Fair enough. I just wondered if there were differences in supported values between projects